audio/pcaudiolib/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                pcaudiolib
+github.setup        espeak-ng pcaudiolib 1.1
+version             1.1
+revision            1
+categories          audio
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3+
+
+description         Portable C Audio Library
+
+long_description    Portable C Audio Library
+
+homepage            https://github.com/espeak-ng/pcaudiolib
+distname            ${version}
+checksums           rmd160  f68876d3346734e993b9d5ebfc3ef48de9161851 \
+                    sha256  449d69c3fd4ed0136d357fd42a73accb276da9b35a19258a379e3ac7ac88c148 \
+                    size    34082
+
+depends_lib         port:automake \
+                    port:autoconf \
+                    port:libtool \
+                    port:pkgconfig
+
+configure.cmd        ./autogen.sh && ./configure
+build.args-append    prefix=${prefix}
+destroot.args        {*}${build.args}
+
+test.run            yes
+test.cmd            make check
+test.target
```

Comment:
Don't put the test target into test.cmd; put it into test.target. The default for test.cmd is make so you can remove the test.cmd line and just set `test.target check`.